HHS/Head-Start-TTADP
# [Prod] Fix session sorting, clean up target populations, add admin form for creating goals

## Description of change

**TTAHUB-2150** Fix session report sorting.
**TTAHUB-2241** Clean up target populations for AR and TR.
**TTAHUB-2179** Batch score and cache historical goals.
**TTAHUB-2268** More robust error logging
**TTAHUB-2255** Multi-line SQL queries mess up logging aggregator Kibana and NewRelic
**TTAHUB-2176** SERVICE:ACTIVITY_REPORTS: invalid input value for enum "enum_Objectives_suspendReason"
**TTAHUB-2288** Fix ActivityReportGoalFieldResponse update bug

### 2109
Add admin form for creating multi-recipient goals

### 2147
Make certain user data available to Google Analytics (user ID and region ID)

## How to test

**TTAHUB-2150** Session sorting on TR landing, complete event, and view event screen should now be sorted.
**TTAHUB-2241** See jira ticket for complete details.
**TTAHUB-2179** After 3am, check DB on prod and ensure SimScoreGoalCache table is populated.
**TTAHUB-2288** Can change goal response

### 2109
Create a multirecipient goal from the admin panel. Test that the validation, creating reports, etc work as expected.

### 2147
Has to be tested via Google Tag Assistant, reach out to Matt for more information

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2150
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2241
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2179
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2109
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2147
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2176
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2255
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2268
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2288

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
